packages/common/src/downloads/_internal/file-url-fetchers/fetchExportImageDownloadFile.ts

```diff
@@ -0,0 +1,109 @@
+import { request } from "@esri/arcgis-rest-request";
+import {
+  DownloadOperationStatus,
+  IFetchDownloadFileOptions,
+  IFetchDownloadFileResponse,
+  ServiceDownloadFormat,
+} from "../../types";
+import HubError from "../../../HubError";
+import { getProp } from "../../../objects/get-prop";
+import { IHubEditableContent } from "../../../core/types/IHubEditableContent";
+import { ExportImageFormat } from "../_types";
+
+/**
+ * @private
+ *
+ * Fetches a download file url from an Image Service via the exportImage endpoint.
+ *
+ * @param options options for refining / filtering the resulting download file
+ * @returns a blob containing the download file
+ */
+export async function fetchExportImageDownloadFile(
+  options: IFetchDownloadFileOptions
+): Promise<IFetchDownloadFileResponse> {
+  validateOptions(options);
+
+  const { entity, format, context, progressCallback } = options;
+  progressCallback && progressCallback(DownloadOperationStatus.PENDING);
+
+  const extent = getExportImageExtent(options);
+  const { xmin, xmax, ymin, ymax } = extent;
+  // TODO: do we need to handle latestWkid as well?
```

Comment:
highlighting for visibility

Reply:
To my knowledge, it is safe to omit *latestWkid* and is typically considered as *optional* wherever it is used the ArcGIS ecosystem.

Reply:
Sweet! Either way I have this point encapsulated in https://devtopia.esri.com/dc/hub/issues/10901